src/pages/account/profile/addresses/[id]/index.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  loggingMiddleware,
+  tracingMiddleware,
+  errorMiddleware,
+  recoveryMiddleware,
+  rateLimitingMiddleware,
+} from "@fwl/web/dist/middlewares";
+import { getServerSidePropsWithMiddlewares } from "@fwl/web/dist/next";
+import type { GetServerSideProps } from "next";
+import React from "react";
+import styled from "styled-components";
+import useSWR from "swr";
+
+import { Address } from "@src/@types/profile";
+import { Container } from "@src/components/containers/container";
+import { NeutralLink } from "@src/components/neutral-link/neutral-link";
+import { Layout } from "@src/components/page-layout";
+import { configVariables } from "@src/configs/config-variables";
+import { logger } from "@src/configs/logger";
+import getTracer from "@src/configs/tracer";
+import { authMiddleware } from "@src/middlewares/auth-middleware";
+import { sentryMiddleware } from "@src/middlewares/sentry-middleware";
+
+const AddressOverviewPage: React.FC<{ addressId: string }> = ({
+  addressId,
+}) => {
+  const { error } = useSWR<{ address: Address }, Error>(
+    `/api/profile/get-address?addressId=${addressId}`,
```

Comment:
In think we should treat this kind of routes as REST endpoints, so instead of that, we should have:
```suggestion
    `/api/profile/addresses/${addressId}`,
```
[1/3]